docs/examples/tensorflow/demo/nvutils/image_processing.py

```diff
@@ -137,7 +137,8 @@ def __init__(self,
         with tf.device("/gpu:0"):
             self.images, self.labels = daliop(
                 serialized_pipeline=serialized_pipe,
-                shape=[batch_size, height, width, 3],
+                shapes=[[batch_size, height, width, 3], []],
```

Comment:
If possible, I think that the most idiomatic way would be to pass the shapes as a list of tuples